packages/trader/src/Modules/Trading/Components/Form/Purchase/value-movement.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Icon, Money } from '@deriv/components';
+import { TProposalTypeInfo } from 'Types';
+
+type TValueMovement = {
+    has_error_or_not_loaded: boolean;
+    proposal_info?: TProposalTypeInfo;
+    currency?: string;
+    has_increased?: boolean;
+    is_turbos?: boolean;
```

Comment:
does it have to be optional? its already initialized.

Reply:
i think we better use `Pick<ReturnType<typeof useTraderStore>, 'is_turbos' | 'is_vanilla' | '.......etc'>`from tradestore as much as possible and have a single source or types otherwise its messy and hard to maintain.